js/ui/action_sheet.js

```diff
@@ -191,7 +191,7 @@ const ActionSheet = CollectionWidget.inherit({
 
     _renderPopupTitle: function() {
         this._mapPopupOption('showTitle');
-        this._popup && this._popup._wrapper().toggleClass(ACTION_SHEET_WITHOUT_TITLE_CLASS, !this.option('showTitle'));
+        this._popup && this._popup._$wrapper.toggleClass(ACTION_SHEET_WITHOUT_TITLE_CLASS, !this.option('showTitle'));
```

Comment:
y we use private data here, not public method?

Reply:
actually i'm still not sure should we use `_$wrapper` or `$wrapper()` in files that inherits overlays. There are both ways in different files.

Reply:
Let's use `$wrapper()` everywhere to resolve this inconsistent and decrease probability to broke something in the future if `$wrapper` method will be changed

Reply:
Ok

Reply:
Done